crates/subspace-farmer/src/multi_farming.rs

```diff
@@ -76,23 +77,41 @@ impl MultiFarming {
         options: Options<impl RpcClient>,
         total_plot_size: u64,
         max_plot_size: u64,
+        mock_plot: bool,
```

Comment:
I think plot creation factory method would work better here, then you don't have to have `BenchPlotMock` and similar things polluting library code and can be moved to CLI.